frontend/server/src/main/java/org/pytorch/serve/http/messages/DescribeModelResponse.java

```diff
@@ -142,12 +142,12 @@ public void addWorker(
         workers.add(worker);
     }
 
-    public Metrics getMetrics() {
-        return metrics;
+    public JobQueueStatus getJobQueueStatus() {
+        return jobQueueStatus;
     }
 
-    public void setMetrics(Metrics metrics) {
-        this.metrics = metrics;
+    public void setJobQueueStatus(JobQueueStatus jobQueueStatus) {
+        this.jobQueueStatus = jobQueueStatus;
```

Comment:
TS provide SDK to allow cx to customize the endpoint response. It will break cx's customized plugin when class Metrics and func getMetrics/etMetrics are moved.

Reply:
Updated implementation to retain the `Metrics` class definition as is.